atuin/src/command/client/search/history_list.rs

```diff
@@ -157,20 +164,25 @@ impl DrawState<'_> {
         let time = format_duration(since.try_into().unwrap_or_default());
 
         // pad the time a little bit before we write. this aligns things nicely
-        self.x = PREFIX_LENGTH - 4 - time.len() as u16;
+        self.draw(
+            &SPACES[..((PREFIX_LENGTH - self.x) as usize - 4 - time.len())],
```

Comment:
I hit a regression while testing the `main` branch for another fix. This line produces the following error

```
thread 'main' panicked at atuin/src/command/client/search/history_list.rs:168:20:
byte index 18446744073709551615 is out of bounds of `                `
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

when the <kbd>up</kbd> arrow is pressed immediately after running the previous command in Zsh.

@Nemo157 I guess you know what is happening and how to fix it. Could you take a look at it?

Reply:
Yep, I can add some limits here so it will just not pad if it would overflow.

Reply:
Thanks!